src/main/java/net/greatstart/controllers/UserController.java

```diff
@@ -31,13 +31,13 @@
     private static final String ERROR_PROFILE = "redirect:/";
 
     private UserService userService;
-    private UserConverterService userConverter;
+    private UserProfileMapper userMapper;
     private final Logger logger = LoggerFactory.getLogger(UserController.class);
 
     @Autowired
-    public UserController(UserService userService, UserConverterService userConverterService) {
+    public UserController(UserService userService, UserProfileMapper userMapper) {
         this.userService = userService;
-        this.userConverter = userConverterService;
+        this.userMapper = userMapper;
     }
 
     // TODO error pages (no such user etc.); enable admin user to edit profiles
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)